extensions/in-memory/assetindex-memory/src/test/java/org/eclipse/dataspaceconnector/metadata/memory/InMemoryAssetIndexTest.java

```diff
@@ -253,6 +253,22 @@ void findAll_withSorting() {
         assertThat(index.queryAssets(spec)).containsAll(assets);
     }
 
+    @Test
+    void deleteById_whenPresent_deletes() {
+        var asset = createAsset("foobar");
+        index.accept(asset, createDataAddress(asset));
+        var deletedAsset = index.deleteById(asset.getId());
+
+        assertThat(deletedAsset).isEqualTo(asset);
+        var assets = index.queryAssets(AssetSelectorExpression.Builder.newInstance().whenEquals(Asset.PROPERTY_NAME, asset.getName()).build());
```

Comment:
make this line more readable please - extract var?

Reply:
Please see my change to make it more readable.